libs/meta/include/meta/value_util.hpp

```diff
@@ -86,22 +86,5 @@ constexpr auto Accumulate(F &&f, A &&a, B &&b,
                     std::forward<Seq>(seq)...);
 }
 
-// Simple uses of Accumulate().
-// Since Accumulate() itself operates on packs rather than ranges,
-// there's no special overload to calculate simple sum, like in STL.
-template <typename H, typename... Ts>
-constexpr auto Sum(H &&h, Ts &&... ts) noexcept(
```

Comment:
These two were actually introduce as docs, to demonstrate how `Accumulate` can compensate for the lack of fold-expressions.

Reply:
It's dead code.